### PR TITLE
Bug fixes and code updation

### PR DESCRIPTION
**Pull request by mohdaadilf**
Updated

- file2data_frame function - 'dfs' list initialization was missing; added encoding argument to read json without issues
- main - Renamed new column names to old existing names
- Updated plot title
- Had the script automatically read required json files in the directory instead of user inputting them
- readme - updated to reflect changes in the code

### CODE REVIEW

**Comment by anthonio9** [comment]
I can't believe you actually spent your time to make this old script better, this is amazing! :)
Could tell me how you found it? Is it still useful? I suppose if there's more recognition (and free time on my side) I could get back to it, perhaps even turn it into a proper app with UI.

File: .gitignore
Comment (anthonio9) [inline .gitignore:1]: Good stuff, thanks!

File: spotistat.py [hunk 1]
```diff
@@ -262,6 +283,10 @@ def main(stream_file_list):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) >= 2:
-        print(f'sys: {sys.argv[1:]}')
-        main(sys.argv[1:])
+    all_files = [pos_json for pos_json in os.listdir(os.getcwd())
```
Comment (anthonio9) [inline spotistat.py:286]: Nice approach, I suppose years ago this did not cross my mind (newbie experience), although I remember having clear problems with using multiple data files.

**Review by anthonio9 — APPROVED** [review]
This is really good, clean code and simplification of usage, and apparently I couldn't have done it better myself! :)

**Comment by mohdaadilf** [comment]
Glad you liked the work! This year's Spotify Wrapped had a lot of noise around it, so I decided to find a way to do the Spotify Stat check myself. Googling a way to do that led me to your [reddit post](https://www.reddit.com/r/truespotify/comments/mrmaw3/spotify_stats_explained_analyze_your_streaming/) and briefly went through the videos after which I thought maybe I could help :)

This is a nice script right now, and I definitely think it is useful and also that there might things you can improve, for sure. Glad to have helped :) !

**Comment by anthonio9** [comment]
If you have any ideas on the improvements, or what this should turn into to gather more attention, please let me know or even open an issue about it! :) 
I think this was originally one of the first publicly available scripts of this type, first published in 2020.

**Pull request merged**